.github/workflows/linearizability-nightly.yaml

```diff
@@ -0,0 +1,25 @@
+name: Linearizability Nightly
+on:
+  # schedules always run against the main branch
+  schedule:
+    - cron: '25 9 * * *'
```

Comment:
Can you add a comment explaining the cron.

Reply:
will add on the next PR for testing the release branches